egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -94,6 +94,16 @@ num_splits_asr=1           # Number of splitting for lm corpus.
 
 # Decoding related
 use_k2=false      # Whether to use k2 based decoder
+is_ctc_decoding=false
+inference_with_tlg=false
+if $inference_with_tlg && $is_ctc_decoding ; then
+  echo "Please chose ctc decoding or tlg decoding."
+  echo "Either of them should be false."
+fi
+lang_dir=data/lang_bpe/
```

Comment:
`lang_dir` and `lang_bpe` are not so easy to understand the meaning.
Can you change it more appropriate ones?

My understanding is that this directory is used to store LMs (n-grams) and FSTs.
We may use fst as a keyword, e.g., lang_fst_dir and data/lang_fst

Reply:
If there is some reasons (k2 conventions and so on), we can keep it.
Please let me know your opinions.